frontend/src/lib/candidate/templates/question/QuestionPage.svelte

```diff
@@ -41,7 +37,7 @@
   // Set the selected key on page load, local storage takes precedence
   $: {
     const likertValue = localStorage.getItem(likertLocal);
-    selectedKey = likertValue ? parseInt(likertValue) : answer?.key;
+    selectedKey = likertValue ? parseInt(likertValue) : answer?.value;
```

Comment:
I get a type mismatch here. We could also make this numeric conversion a bit more robust and check for errors in the data by doing something like:

```ts
$: {
    const value = localStorage.getItem(likertLocal) ?? answer?.value;
    if (value != null) {
      const intValue = typeof value === 'number' ? value : parseInt(`${value}`);
      if (`${intValue}` !== `${value}` || !Number.isInteger(intValue)) throw new Error(`Likert question answer value is not an integer: ${value}`);
      selectedKey = intValue;
    }
  }
```